Update vpn-auto-trigger-profile.md (#9476)
MicrosoftDocs/windows-itpro-docs: windows/security/identity-protection/vpn/vpn-auto-trigger-profile.md

- Name-based trigger
- Always On

> >[!NOTE]

Reviewer: ```suggestion
> [!NOTE]
```

> > Auto-triggered VPN connections will not work if Folder Redirection for AppData is enabled. Either Folder Redirection for AppData must be disabled or the auto-triggered VPN profile must be deployed in system context, which changes the path that the rasphone.pbk file is stored.

Reviewer 2: "path to where something is stored"
```suggestion
> Auto-triggered VPN connections will not work if Folder Redirection for AppData is enabled. Either Folder Redirection for AppData must be disabled or the auto-triggered VPN profile must be deployed in system context, which changes the path to where the rasphone.pbk file is stored.
```
Natural daily speech adaption - the phrasing "_the path that something is stored_" is highly unusual.

> >Auto-triggered VPN connections will not work if Folder Redirection for %appdata% (C:\Users\username\AppData\Roaming) is enabled. Either Folder Redirection must be disabled for %appdata% or the auto-triggered VPN profile must be deployed in system context, that changes the path the rasphone.pbk file is stored.

Reviewer: ```suggestion
> Auto-triggered VPN connections will not work if Folder Redirection for %appdata% (C:\Users\username\AppData\Roaming) is enabled. Either Folder Redirection must be disabled for %appdata% or the auto-triggered VPN profile must be deployed in system context, which changes the path that the rasphone.pbk file is stored.
```
